src/codegen/generators/pou_generator.rs

```diff
@@ -61,6 +61,74 @@ pub fn generate_implementation_stubs<'ink>(
     Ok(llvm_index)
 }
 
+pub fn generate_global_constants_for_pou_members<'ink>(
```

Comment:
can we add some basic documentation (where is it getting the POUs from, what does it return)

Reply:
done